exercises/practice/rail-fence-cipher/rail-fence-cipher.spec.js

```diff
@@ -0,0 +1,37 @@
+import { encode, decode } from './rail-fence-cipher';
+
+describe('Rail Fence Cipher', () => {
+  describe('encode', () => {
+    test('encode with two rails', () => {
+      const fence = encode('XOXOXOXOXOXOXOXOXO', 2);
+      expect(fence).toEqual('XXXXXXXXXOOOOOOOOO');
+    });
+    test('encode with three rails', () => {
+      const fence = encode('WEAREDISCOVEREDFLEEATONCE', 3);
+      expect(fence).toEqual('WECRLTEERDSOEEFEAOCAIVDEN');
+    });
+    test('encode with ending in the middle', () => {
+      const fence = encode('EXERCISES', 4);
+      expect(fence).toEqual('ESXIEECSR');
+    });
+  });
+  describe('decode', () => {
+    test('decode with three rails', () => {
+      const fence = decode('TEITELHDVLSNHDTISEIIEA', 3);
+      expect(fence).toEqual('THEDEVILISINTHEDETAILS');
+    });
+    test('decode with five rails', () => {
+      const fence = decode('EIEXMSMESAORIWSCE', 5);
+      expect(fence).toEqual('EXERCISMISAWESOME');
+    });
+    test('decode with six rails', () => {
+      const fence = decode(
+        '133714114238148966225439541018335470986172518171757571896261',
```

Comment:
I would recommend to use a variable for the input string here and in line 33 instead of having the multi line function call.